cclib/parser/turbomoleparser.py

```diff
@@ -208,6 +228,53 @@ def extract(self, inputfile, line):
             self.set_attribute('vibirs', vibirs)
             self.set_attribute('vibdisps', vibdisps)
 
+        # In this section we are parsing mocoeffs and moenergies from
+        # the files like: mos, alpha and beta.
+        # $scfmo    scfconv=6   format(4d20.14)

```

Comment:
The presence of `format(4d20.14)` makes me wonder if Turbomol has an option to change the format of the orbitals block. Is there anything in the documentation that talks about this?

Reply:
I didn't find it as such. What does this mean exactly?

Reply:
4 entries per column, and each entry has 20 characters with 14 decimal places. I'm guessing the d means using D instead of E for the exponent.

Reply:
If we are comfortable adding a dependency, I've used [fortranformat](https://pypi.org/project/fortranformat/) in the past and really like it.

Reply:
I dunno, it doesn't feel like a compelling reason to add it as a dependency. We can easily add the functionality we need in one function, no?